addons/mail/static/src/new/messaging_menu/messaging_menu.xml

```diff
@@ -96,9 +96,9 @@
     </div>
     <div t-if="store.isSmall" class="o-mail-messaging-menu-navbar d-flex border-top bg-view shadow-lg w-100">
         <div t-foreach="tabs" t-key="tab.id" t-as="tab" class="o-mail-messaging-menu-tab d-flex flex-column align-items-center flex-grow-1 flex-basis-0 p-2" t-att-class="{
-            'text-primary fw-bolder': state.tab === tab.id,
+            'text-primary fw-bolder o-active': state.tab === tab.id,
             'border-end': !tab_last,
-        }" t-on-click="() => this.onClickNavTab(tab.id)">
+        }" t-att-data-tab-id= "tab.id" t-on-click="() => this.onClickNavTab(tab.id)">
```

Comment:
Remove `t-att-data-tab-id`. Instead on relying on that in the test, use visual clues, e.g. the label of the button.